docs/book/src/clusterctl/commands/move.md

```diff
@@ -58,6 +58,16 @@ make it clear those operation have the same limitations of the move command.
 
 </aside>
 
+<aside class="note warning">
```

Comment:
Given that this note is for the provider implementers, have you considered adding it to https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#move?

Reply:
I added the same note there. See #a8822b0ac